services/movie.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import QuerySet
+
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: int = None,
+        actors_ids: int = None

```

Comment:
The parameters `genres_ids` and `actors_ids` should be lists of integers, not single integers. Update the type hints to `List[int]` and ensure the function handles lists correctly.